syntax = "proto2";

message DownloadResp {
	required int64 state = 1;
}
